src/components/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+const Articles = ({ article }) => {
+  const myDate = new Date(article.created_at);
+  const day = myDate.getDate();
+  const month = myDate.toLocaleString("default", { month: "long" });
+  const year = myDate.getFullYear();
+  const date = `${day}/${month}/${year}`;
+  return (
+    <div className="ui card">
```

Comment:
Try to avoid using div where possible, what can you use to make it more accessible for the user?